tests/data/events_with_types.yaml

```diff
@@ -18,8 +18,10 @@ core:
     extra_keys:
       key1:
         description: "This is key one"
+        type: string
```

Comment:
I'm not seeing where this metric is used in any of the tests, only the preference_toggled one seems to be checked by `assert`s